COMP120_Evaluation.tex

```diff
@@ -0,0 +1,45 @@
+% Please do not change the document class
+\documentclass{scrartcl}
+
+% Please do not change these packages
+\usepackage[hidelinks]{hyperref}
+\usepackage[none]{hyphenat}
+\usepackage{setspace}
+\doublespace
+
+% You may add additional packages here
+\usepackage{amsmath}
+
+% Please include a clear, concise, and descriptive title
+\title{Evaluation}
+
+% Please do not change the subtitle
+\subtitle{COMP120 - Evaluation}
+
+% Please put your student number in the author field
+\author{James (JH182233) 1506530}
+
+\begin{document}
+
+\maketitle
+
+\section{Time Keeping}
+
+Over the course I found that I kept getting distracted from my projects. For me the most significant issue was not practising the practical side of programming, at first I felt lost, this might be due to the fact that i had been out of education for an extended period of time. Unlike being employed, education requires you to demonstrate your own timekeeping more strictly than employment would. Having experienced university I now realise I have not sufficiently developed my ability to use my free time efficiently. I will schedule my time and note it to keep track of my progress, by creating a daily timetable that will account for programming practice and leisure.
```

Comment:
I would suggest seeking support from the [ASK team](http://ask.fxplus.ac.uk/). Further to this, you need to show how your proposed action is SMART.